File: old_version/npcapp/src/main/java/com/UpwardEdge/entities/Npc.java
package com.UpwardEdge.entities;

import java.util.ArrayList;
import java.util.List;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.OneToMany;
import javax.persistence.SequenceGenerator;
import javax.persistence.Table;

@Entity
@Table(name="NPC")
public class Npc {

	@Id
	@Column(name="npc_id")
	@GeneratedValue(generator="npc_sequence", strategy=GenerationType.SEQUENCE)
	@SequenceGenerator(name="npc_sequence", sequenceName="npc_sequence", allocationSize=1)
	private int NpcId;
	
	private String name;
	private String campaign;
	private String race;
	private String occupation;
	private String country;
	private String town;
	@Column(name="physical_desc")
	private String physicalDesc;
	@Column(name="voice_desc")
	private String voiceDesc;
	@Column(name="personality_desc")
	private String personalityDesc;
	private String organization;
	private String comments;

	public Npc(int npcId, String name, String campaign, String race, String occupation, String country, String town,
			String physicalDesc, String voiceDesc, String personalityDesc, String organization, String comments) {
		super();
		NpcId = npcId;
		this.name = name;
		this.campaign = campaign;
		this.race = race;
		this.occupation = occupation;
		this.country = country;
		this.town = town;
		this.physicalDesc = physicalDesc;
		this.voiceDesc = voiceDesc;
		this.personalityDesc = personalityDesc;
		this.organization = organization;
		this.comments = comments;
	}

	public Npc() {
		super();
		// TODO Auto-generated constructor stub
	}

	public int getNpcId() {
		return NpcId;
	}

	public void setNpcId(int npcId) {
		NpcId = npcId;
	}

	public String getName() {
		return name;
	}

	public void setName(String name) {
		this.name = name;
	}

	public String getCampaign() {
		return campaign;
	}

	public void setCampaign(String campaign) {
		this.campaign = campaign;
	}

	public String getRace() {
		return race;
	}

	public void setRace(String race) {
		this.race = race;
	}

	public String getOccupation() {
		return occupation;
	}

	public void setOccupation(String occupation) {
		this.occupation = occupation;
	}

	public String getCountry() {
		return country;
	}

	public void setCountry(String country) {
		this.country = country;
	}

	public String getTown() {
		return town;
	}

	public void setTown(String town) {
		this.town = town;
	}

	public String getPhysicalDesc() {
		return physicalDesc;
	}

	public void setPhysicalDesc(String physicalDesc) {
		this.physicalDesc = physicalDesc;
	}

	public String getVoiceDesc() {
		return voiceDesc;
	}

	public void setVoiceDesc(String voiceDesc) {
		this.voiceDesc = voiceDesc;
	}

	public String getPersonalityDesc() {
		return personalityDesc;
	}

	public void setPersonalityDesc(String personalityDesc) {
		this.personalityDesc = personalityDesc;
	}

	public String getOrganization() {
		return organization;
	}

	public void setOrganization(String organization) {
		this.organization = organization;
	}

	public String getComments() {
		return comments;
	}

	public void setComments(String comments) {
		this.comments = comments;
	}

	@Override
	public String toString() {
		return "Npc [NpcId=" + NpcId + ", name=" + name + ", campaign=" + campaign + ", race=" + race + ", occupation="
				+ occupation + ", country=" + country + ", town=" + town + ", physicalDesc=" + physicalDesc
				+ ", voiceDesc=" + voiceDesc + ", personalityDesc=" + personalityDesc + ", organization=" + organization
				+ ", comments=" + comments + "]";
	}

	
}

File: old_version/npcapp-ui/src/app/beans/Npc.ts
export class Npc {
    private id: number;
    private name: string;
    private campaign: string;
    private race: string;
    private occupation: string;
    private country: string;
    private town: string;
    private physicalDesc: string;
    private voiceDesc: string;
    private personalityDesc: string;
    private organization: string;
    private comments: string;

    /**
     * Getter $id
     * @return {number}
     */
	public get $id(): number {
		return this.id;
	}

    /**
     * Setter $id
     * @param {number} value
     */
	public set $id(value: number) {
		this.id = value;
	}

    /**
     * Getter $name
     * @return {string}
     */
	public get $name(): string {
		return this.name;
	}

    /**
     * Setter $name
     * @param {string} value
     */
	public set $name(value: string) {
		this.name = value;
	}

    /**
     * Getter $campaign
     * @return {string}
     */
	public get $campaign(): string {
		return this.campaign;
	}

    /**
     * Setter $campaign
     * @param {string} value
     */
	public set $campaign(value: string) {
		this.campaign = value;
	}

    /**
     * Getter $race
     * @return {string}
     */
	public get $race(): string {
		return this.race;
	}

    /**
     * Setter $race
     * @param {string} value
     */
	public set $race(value: string) {
		this.race = value;
	}

    /**
     * Getter $occupation
     * @return {string}
     */
	public get $occupation(): string {
		return this.occupation;
	}

    /**
     * Setter $occupation
     * @param {string} value
     */
	public set $occupation(value: string) {
		this.occupation = value;
	}

    /**
     * Getter $country
     * @return {string}
     */
	public get $country(): string {
		return this.country;
	}

    /**
     * Setter $country
     * @param {string} value
     */
	public set $country(value: string) {
		this.country = value;
	}

    /**
     * Getter $town
     * @return {string}
     */
	public get $town(): string {
		return this.town;
	}

    /**
     * Setter $town
     * @param {string} value
     */
	public set $town(value: string) {
		this.town = value;
	}

    /**
     * Getter $physicalDesc
     * @return {string}
     */
	public get $physicalDesc(): string {
		return this.physicalDesc;
	}

    /**
     * Setter $physicalDesc
     * @param {string} value
     */
	public set $physicalDesc(value: string) {
		this.physicalDesc = value;
	}

    /**
     * Getter $voiceDesc
     * @return {string}
     */
	public get $voiceDesc(): string {
		return this.voiceDesc;
	}

    /**
     * Setter $voiceDesc
     * @param {string} value
     */
	public set $voiceDesc(value: string) {
		this.voiceDesc = value;
	}

    /**
     * Getter $personalityDesc
     * @return {string}
     */
	public get $personalityDesc(): string {
		return this.personalityDesc;
	}

    /**
     * Setter $personalityDesc
     * @param {string} value
     */
	public set $personalityDesc(value: string) {
		this.personalityDesc = value;
	}

    /**
     * Getter $organization
     * @return {string}
     */
	public get $organization(): string {
		return this.organization;
	}

    /**
     * Setter $organization
     * @param {string} value
     */
	public set $organization(value: string) {
		this.organization = value;
	}

    /**
     * Getter $comments
     * @return {string}
     */
	public get $comments(): string {
		return this.comments;
	}

    /**
     * Setter $comments
     * @param {string} value
     */
	public set $comments(value: string) {
		this.comments = value;
	}
}
